### PR TITLE
Add TcPdfOutput

### LINKED ISSUE #50: New library to generate PDF documents using TCPDF
In my point of view, this library must not depend on any library to output payment slips. Therefore, #32 should never be merged.

I have created a new library compatible with `Sprain\SwissQrBill\PaymentPart\Output\OutputInterface` and `Sprain\SwissQrBill\PaymentPart\Output\AbstractOutput` to extend this one in order to generate mass-mailing QR-Bill documents using TCPDF library.

https://github.com/sdespont/php-swiss-qr-bill-tcpdf

@sprain What do you think about it? 
@Franco2911, this could be useful for you

### PR DESCRIPTION
**Pull request by sprain**
Resolves #50

* Integrates `TcPdfOutput` into the library (as currently available [as a separate project](https://github.com/sdespont/php-swiss-qr-bill-tcpdf))
* Improves setup of examples accordingly

**Todos**

- [x] Make sure tests pass reliably
- [x] Fix wrongly placed placeholders (see [this file for example](https://github.com/sprain/php-swiss-qr-bill/blob/49c59f0fc7d5414223e503c6ce78f78697fc06da/tests/TestData/TcPdfOutput/qr-additional-information.pdf))

### CODE REVIEW

**Comment by sdespont** [comment]
Is it normal that some reference files have the placeholders **and** amounts ? Like qr-additional-information.pdf for example?

File: src/PaymentPart/Output/TcPdfOutput/TcPdfOutput.php [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+            );
+            $this->tcPdf->Ln($isReceiptPart ? self::TCPDF_9PT : self::TCPDF_11PT);
+        }
+
+        if ($element instanceof Placeholder) {
+            $this->tcPdf->SetLineStyle(array('width' => 0.3, 'dash' => 0, 'color' => array(0, 0, 0)));
+
+            $lineLength = 3;
+
+            // Not implemented
```
Comment (sdespont) [inline src/PaymentPart/Output/TcPdfOutput/TcPdfOutput.php:249]: In fact, it is implemented now :o)

**Comment by sprain** [comment]
> Is it normal that some reference files have the placeholders and amounts ? 

It‘s a valid use-case and therefore sprinkled into the tests :)

**Comment by sprain** [comment]
> Tell me if you need help

Thanks! I am currently busy with other things. I'll get back to this as soon as I can.

**Comment by sprain** [comment]
@sdespont I fixed and simplified the handling of placeholders by using the official graphics. Any comments by you?
https://github.com/sprain/php-swiss-qr-bill/pull/53/commits/ed95896f5bc1c07709fa0c2fb2ca65e7f66f2bd9#diff-50f34ff1128cf60b1ac0ae7ad32d282b

**Comment by sprain** [comment]
@sdespont IMO this is now ready to merge. But I'd be happy to get your review first.

**Comment by sdespont** [comment]
@sprain OK for me, you can merge. Thanks for the integration.

**Comment by sprain** [comment]
v2.11 it is 😊
https://github.com/sprain/php-swiss-qr-bill/releases/tag/v2.11

**Pull request merged**